database/cassandra/cassandra.go

```diff
@@ -130,9 +145,17 @@ func (p *Cassandra) Run(migration io.Reader) error {
 	}
 	// run migration
 	query := string(migr[:])
-	if err := p.session.Query(query).Exec(); err != nil {
-		// TODO: cast to Cassandra error and get line number
-		return database.Error{OrigErr: err, Err: "migration failed", Query: migr}
+	matches := regexp.MustCompile(`(?m:;$)`).Split(query, -1)
```

Comment:
It didn't execute multiple queries before?

Reply:
I just submitted #280 (but didn't know about this PR until now) which has an opt-in when it comes to multi-statement support to avoid the case of
```cql
INSERT INTO codesamples(code) VALUES ('import a;
import b;');
```
Might be worth considering.

Reply:
No, unfortunately not.